default.nix

```diff
@@ -18,14 +18,23 @@ with pkgs.lib;
 with pkgs.haskell.lib;
 
 let
+  doHaddockHydra = drv: overrideCabal drv (attrs: {
+    doHaddock = true;
+    postInstall = ''
+      ${attrs.postInstall or ""}
+      mkdir -pv $doc/nix-support
+      tar -czvf $doc/${attrs.pname}-docs.tar.gz -C $doc/share/doc/html .
+      echo "file binary-dist $doc/${attrs.pname}-docs.tar.gz" >> $doc/nix-support/hydra-build-products
+      echo "report ${attrs.pname}-docs.html $doc/share/doc/html index.html" >> $doc/nix-support/hydra-build-products
```

Comment:
Do we need both this and the `docs/language-plutus-core.nix` below? The latter seems unnecessary to me, unless I'm missing something.

Reply:
yeah, I missed that when cleaning up. It's not even referenced in default.nix anymore.